spatialmath/geometry.go

```diff
@@ -66,31 +68,31 @@ type GeometryConfig struct {
 }
 
 // NewGeometryConfig creates a config for a Geometry from an offset Pose.
-func NewGeometryConfig(gc Geometry) (*GeometryConfig, error) {
+func NewGeometryConfig(g Geometry) (*GeometryConfig, error) {
```

Comment:
Made some changes in this file to rename variables to be more accurate as we no longer have the `gc` (GeometryCreator) type